rare/shared/image_manager.py

```diff
@@ -155,7 +155,7 @@ def __prepare_download(self, game: Game, force: bool = False) -> Tuple[List, Dic
             json_data = json.load(open(self.__img_json(game.app_name), "r"))
 
         # lk: fast path for games without images, convert Rare's logo
-        if not game.metadata["keyImages"]:
```

Comment:
Why not game.metadata.get()?

Reply:
Because I was not aware of the `get()` method for dictionaries 😅. But also because at that point we don't need the value of it, just to check for its existence.

Reply:
But you read the value in the second condition?

Reply:
Maybe it would be, you are correct. It is just preference I guess. Conceptually I would use `get()` in the context of assigning the return value to a variable.